components/CocktailsOverview/CocktailsOverview.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable react/jsx-key */
+import Image from "next/image";
+import useSWR from "swr";
+import styled from "styled-components";
+
+const URL = "https://www.thecocktaildb.com/api/json/v1/1/search.php?s=cocktail";
+
+export default function CocktailsOverview({}) {
+  const fetcher = (URL) => fetch(URL).then((response) => response.json());
+  const { data, error } = useSWR(URL, fetcher);
+  const drinks = data?.drinks;
+
+  if (error) return <div>Failed to load</div>;
+  if (!data) return <div>Loading...</div>;
+
+  return (
+    <Container>
+      <ul>
+        {drinks?.map(({ idDrink, strDrink, strDrinkThumb }) => (
+          <ListItem>
+            <li key={idDrink}>
+              <h2>{strDrink}</h2>
+              <Image
+                src={strDrinkThumb}
+                alt={strDrink}
+                height={100}
+                width={100}
+              />
+            </li>
+          </ListItem>
```

Comment:
```suggestion
          <ListItem key={idDrink}>
            <h2>{strDrink}</h2>
            <Image
              src={strDrinkThumb}
              alt={strDrink}
              height={100}
              width={100}
            />
          </ListItem>
```
- The `key`should always be part of the parent element inside the `map` method.
- You created a nested `<li>` inside another one because `ListItem` is already a `<li>` tag